### PR TITLE
Module 5

### PR DESCRIPTION
**Pull request by JoeBawn**
Please make sure you are raising this PR against your own repository and not the original. If it says https://github.com/CorndelWithSoftwire in the search bar right now, you are in the wrong place!

Please also don't forget to submit the PR's url to Aptem afterwards (by clicking on the corresponding exercise component, pasting the url in the textbox that appears and then pressing the finish button).

### CODE REVIEW

**Comment by JoeBawn** [comment]
Merge Module 5 - Rework required for pyproject.toml file to add gunicorn back in due to issues with merging pull request.

**Review by JackMead — COMMENTED** [review]
I'll leave the full feedback on [the updated PR](https://github.com/JoeBawn/DevOps-Course-Starter/pull/8), but figured I'd leave line-by-line thoughts here for simplicity

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+WORKDIR /temp
+COPY ./poetry.toml /temp
+COPY ./pyproject.toml /temp
+RUN pip install poetry && poetry install
+
+FROM base as production
+RUN apt-get update
+COPY ./todo_app /temp/todo_app
+EXPOSE 5000
+ENTRYPOINT $(poetry env info --path)/bin/gunicorn --error-logfile /temp/error.log -b 0.0.0.0:5000 "todo_app.app:create_app()"
```
[the remark this answers is not in the document]
Comment (JoeBawn) [reply Dockerfile:12]: This has now been updated.

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,18 @@
+FROM python:3 as base
+
+WORKDIR /temp
+COPY ./poetry.toml /temp
+COPY ./pyproject.toml /temp
+RUN pip install poetry && poetry install
+
+FROM base as production
+RUN apt-get update
```
[the remark this answers is not in the document]
Comment (JoeBawn) [reply Dockerfile:9]: This has now been updated.

**Comment by JoeBawn** [comment]
Thanks for the feedback Jack. Have made suggested changes.

**Pull request merged**